src/app.js

```diff
@@ -0,0 +1,29 @@
+import './assets/scss/app.scss';
+import $ from 'cash-dom';
+
+
+export class App {
+  initializeApp() {
+    let self = this;
+
+    $('.load-username').on('click', function (e) {
+      let userName = $('.username.input').val();
```

Comment:
```suggestion
      const userName = $('.username.input').val();
```

you don't rewrite variable, so you can use const